src/Microsoft.ML.Core/Data/KeyType.cs

```diff
@@ -59,6 +59,13 @@ public static bool IsValidDataType(Type type)
         /// </summary>
         public ulong Count { get; }
 
+        /// <summary>
+        /// Determine if a DataViewType object is equal to another KeyType instance.
```

Comment:
>KeyType [](start = 67, length = 7)

I might reverse this, so, "if this `KeyType` instance is equal to another `DataViewType` instance." SInce *this* is assuredly a key type, and the other thing is assuredly a `DataView` Type. Note also, I might suggest being consistent in this sentence between the usage of instance and object. And the use of "this" instead of "a". Anyway, these latter points are not really big deals.